src/detections/filters/accordion-details/cameras/select.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Select as MuiSelect } from '@material-ui/core'
+import { useSelector } from 'react-redux'
+
+import { MenuItem } from 'detections/filters/accordion-details/cameras/menu-item'
+
+export const Select = () => {
+  const { items } = useSelector(({ cameras }) => cameras.get.data)
```

Comment:
Let's make sure to fix this :)))
![image](https://user-images.githubusercontent.com/23569938/129182011-0b605996-a5a4-4eeb-a792-117c34a18609.png)

Reply:
done